app/assets/javascripts/app/collections/comments.js

```diff
@@ -12,12 +12,17 @@ app.collections.Comments = Backbone.Collection.extend({
 
   make : function(text) {
     var self = this;
-    var comment = new app.models.Comment({ "text": text });
+    var comment = new app.models.Comment({ "text": text },{ post: this.post});
```

Comment:
- There should be no space after '{'
- There should be no space before '}'
- A space is required after ','.